frontend/src/components/AnalyticsDrawer/CensusAnalytics.tsx

```diff
@@ -11,8 +14,23 @@ export let CensusAnalytics = withStore(
 
   let selectedServiceAreas = store.get('selectedServiceAreas') ? store.get('selectedServiceAreas')! : store.get('serviceAreas')
   let selectedCensusCategory = store.get('selectedCensusCategory')
+  let populationByAdequacy = summaryStatisticsByServiceArea(selectedServiceAreas, store)
+  let totalPopulation = populationByAdequacy.reduce(function (a, b) { return a + b }, 0)
+  let totalProviders = store.get('providers').length
 
   return <div className='CensusAnalytics'>
+    <StatsBox className='HighLevelStats -withFixedColumnns' withBorders>
+      <tr>
+        <th>Total Population</th>
+        <th>Providers</th>
+        <th>Provider Ratio</th>
+      </tr>
+      <tr>
+        <td>{formatNumber(totalPopulation)}</td>
+        <td>{formatNumber(totalProviders)}</td>
+        <td>1 : {formatNumber(totalPopulation / totalProviders)}</td>
```

Comment:
this worries me a little bit just because we have single providers potentially with multiple specialties and multiple addresses

Reply:
@viviandien what are your thoughts on this? We can also keep population and providers for now and think a bit more about the ratio